hdtv/app.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
> python2 is not supported anymore and python3 is the default name of the python3 executable on any "normal" system

I kind of disagree with that statement - a "normal" system would have `python` symlinked to `python3`, and python2 not installed at all.

Reply:
I disagree, take for a example a standard Ubuntu (AFAIK one of the most if not the one most used distribution) install in any version:
Ubuntu versions that don't ship with python2 anymore (like Ubuntu 22.04) won't have python3 symlinked to python by default.
Older Ubuntu versions that still shipped with python2 (like Ubuntu 20.04 which as a LTS release will still be supported till 2030!), have python symlinked to python2, and here AFAIK it is also not recommended to change this symlink as potentially other older dependencies just expect it to refer to python2.

Also from the [Ubuntu wiki](https://wiki.ubuntu.com/Python/3):

> /usr/bin/python will point to Python 3. No, this is not going to happen (unless [PEP 394](http://legacy.python.org/dev/peps/pep-0394/) advocates otherwise, which is doubtful for the foreseeable future). /usr/bin/python and /usr/bin/python2 will point to Python 2.7 and /usr/bin/python3 will point to the latest supported Python 3 version.

Most user will just set an alias in their shell (e.g. in the .bash.rc) for python to refer to python3, but this is ignored by /usr/bin/env.

So with a default Ubuntu install you cannot run these hdtv scripts like app.py immediately, instead you have to either call python3 explicitly or mess with your env settings/symlinks.
I guess it will be the same for Debian, MINT, and many other distros (except maybe for arch linux).

And hdtv only supports python3 anyway, so why not explicitly require it instead of implicitly requiring the user to change their env?
[PEP 394](https://peps.python.org/pep-0394/) requires python3 to be found as python3 in the PATH of Unix systems, while for python it is up to the distro whether it points to python2, python3 or is not existing at all.
So requesting python3 from env will not pose any problems, but may make running hdtv from source easier for some users.

Reply:
Alright. I despise the overall situation, but agree that python3 allows for the best compatibility.